actionmailer/test/fixtures/another.path/base_mailer/welcome

```diff
@@ -0,0 +1 @@
+Welcome from another path
```

Comment:
please add new lines in end of file.

Reply:
Yes, it's better to add newline and change the test case since I believe that's generally how most editors are configured (with newline at eof).